kornia/losses/dice.py

```diff
@@ -48,8 +54,7 @@ def dice_loss(input: torch.Tensor, target: torch.Tensor, eps: float = 1e-8) -> t
         >>> output = dice_loss(input, target)
         >>> output.backward()
     """
-    if not isinstance(input, torch.Tensor):
-        raise TypeError(f"Input type is not a torch.Tensor. Got {type(input)}")
+    KORNIA_CHECK_IS_TENSOR(input, f"Input type is not a torch.Tensor. Got {type(input)}")
```

Comment:
we already have this msg on kornia check 😄 
```suggestion
    KORNIA_CHECK_IS_TENSOR(input)
```